app/models/form/sales/pages/buyer2_relationship_to_buyer1_yes_no.rb

```diff
@@ -0,0 +1,23 @@
+class Form::Sales::Pages::Buyer2RelationshipToBuyer1YesNo < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @id = "buyer_2_relationship_to_buyer_1"
+    @copy_key = "sales.household_characteristics.relat2.buyer"
+    @depends_on = [
+      {
+        "joint_purchase?" => true,
+        "buyer_has_seen_privacy_notice?" => true,
+      },
+      {
+        "joint_purchase?" => true,
+        "buyer_not_interviewed?" => true,
+      },
+    ]
+  end
```

Comment:
I dont think `buyer_has_seen_privacy_notice` and `buyer_not_interviewed` is needed anymore if this page only exists on 2025. I think it's covered by privacy notice and interview questions being in the setup section

Reply:
Updated to depend just on joint_purchase? => true